src/app/components/License/License.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Html5Entities } from 'html-entities';
+import * as DS from '@nypl/design-system-react-components';
+import { Link } from 'react-router';
+import { detailDefinitionLabels } from '../../constants/labels';
+import {
+    unique, flattenDeep, isEmpty, uniqueAndSortByFrequency,
+} from '../../util/Util';
+
+export const License = () => {
+    return (<DS.Container>
```

Comment:
No breadcrumb! Without it there is no obvious link back to ResearchNow

Reply:
d'oh!  Yes I will add this!